projects/packages/videopress/src/client/utils/video-chapters/validate-chapters/index.ts

```diff
@@ -0,0 +1,53 @@
+import type { VideoPressChapter } from '../types';
+
+/**
+ * Calculates the number of seconds from a timestamp in the format hh:mm:ss
+ *
+ * @param {string} time - The timestamp
+ * @returns {number}    - The number of seconds
+ */
+function getSeconds( time: string ): number {
+	const timeSections = time.split( ':' );
+
+	return (
+		parseInt( timeSections[ 0 ] ) * 3600 +
+		parseInt( timeSections[ 1 ] ) * 60 +
+		parseInt( timeSections[ 2 ] )
+	);
+}
+
+/**
+ * Validates the generated chapters against UX restrictions
+ *
+ * @param {VideoPressChapter[]} chapters - The chapters to be validated
+ * @returns {boolean} - Whether the generated chapters are valid or not
+ */
+export default function validateChapters( chapters: VideoPressChapter[] ) {
+	if ( ! chapters || chapters.length === 0 ) {
+		return false;
+	}
+
+	// The first timestamp should be 00:00:00
+	if ( chapters[ 0 ].startAt !== '00:00:00' ) {
```

Comment:
The documentation states:

> Make sure that the first timestamp starts with 00:00.

So I think we should allow the `mm:ss` format, as well as the `hh:mm:ss`, maybe this way:

```suggestion
	if ( chapters[ 0 ].startAt !== '00:00' && chapters[ 0 ].startAt !== '00:00:00' ) {
```

Reply:
Never mind. I was checking the extraction logic and it does the magic for us, transforming any valid-ish format of `hh:mm:ss` or `mm:ss` or `m:ss` to the right `hh:mm:ss` format so here we are already dealing with clean values 🏆